lib/action-ros2-ci.js

```diff
@@ -32,6 +32,12 @@ function run() {
             };
             yield exec.exec("bash", ["-c",
                 "curl https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos | vcs import src/"], options);
+            // If the package under tests is part of ros2.repos, remove it first.
+            // We do not want to allow the "default" head state of the package to
+            // to be present in the workspace, and colcon will fail stating it found twice
+            // a package with an identical name.
+            yield exec.exec("bash", ["-c",
```

Comment:
thinking out loud - should we modify the repos file before importing it, instead of cloning and then deleting the pkg we're working on?

Reply:
That's a small speed decrease in front of the whole compilation time (this action is CI, so we're talking 10+ minutes). `vcs` does not have an option to do that, so we'd pull a dependency on the internal YAML format there. So not sure we'd gain that much from doing that here.

Reply:
Ok sounds reasonable